internal/runner/runner_test.go

```diff
@@ -45,17 +46,32 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func TestRunNoUpdatesCase(t *testing.T) {
+	inBytes, err := ioutil.ReadFile("testdata/no_direct_updates.json")
+	if err != nil {
+		t.Errorf("Failed to read input file: %s", err)
+	}
+	in := bytes.NewBuffer(inBytes)
+	var out bytes.Buffer
+	err = runner.Run(in, &out, true, false, false, runner.StyleDefault)
+	if err != nil {
+		t.Errorf("Error should be nil, got %s", err)
+	}
+	if out.Len() != 0 {
+		t.Errorf("Wanted an empty output, got \n%q", out.String())
+	}
+}
```

Comment:
```suggestion
func TestRunNoUpdatesCase(t *testing.T) {
	inBytes, err := ioutil.ReadFile("testdata/no_direct_updates.json")
	if err != nil {
		t.Errorf("Failed to read input file: %s", err)
	}

	in := bytes.NewBuffer(inBytes)

	var out bytes.Buffer

	err = runner.Run(in, &out, true, false, false, runner.StyleDefault)
	if err != nil {
		t.Errorf("Error should be nil, got %s", err)
	}

	if out.Len() != 0 {
		t.Errorf("Wanted an empty output, got \n%q", out.String())
	}
}
```

just to fix wsl linter